addons/account/tests/test_account_move_entry.py

```diff
@@ -1043,3 +1043,26 @@ def test_change_journal_account_move(self):
             self.assertEqual(move_form.name, 'MISC/2021/10/0001')
             move_form.journal_id, journal = journal, move_form.journal_id
             self.assertEqual(move_form.name, 'AJ/2021/10/0001')
+
+    def test_manually_modifying_taxes(self):
```

Comment:
Can't find what changed it, but your test is passing without your changes in the code

Reply:
I guess this test case is too simple and does not represent the case in the ticket enough. I'll add another one, because the original issue is not yet solved without this change.

Reply:
@william-andre I changed the test so it does fail when my change is not there.